integration-tests/broker_setup_test.go

```diff
@@ -0,0 +1,38 @@
+package integration_tests
+
+import (
+	"encoding/json"
+	testframework "github.com/cloudfoundry/cloud-service-broker/brokerpaktestframework"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var mockTerraform testframework.TerraformMock
+var broker *testframework.TestInstance
+
+var BrokerGCPProject = "broker-gcp-project"
+var BrokerGCPCreds = "broker-gcp-creds"
+
+var _ = BeforeSuite(func() {
```

Comment:
Can this entire file be part of the `integration-tests/integration_tests_suite_test.go`. I don't know if that is common, but I would expect all the suite level setup to be there (ignore it if you think that's just my style)